src/app/pages/apps/components/app-detail-view/app-detail-view.component.html

```diff
@@ -15,9 +15,8 @@
           mat-button
           color="primary"
           [ixTest]="[app?.name, 'install']"
-          [disabled]="!app"
-          (click)="installButtonPressed()"
-        >{{ 'Install' | translate }}</a>
+          (click)="wasPoolSet ? installButtonPressed() : onChoosePool()"
```

Comment:
Let's change  **On**ChoosePool to `showChoosePoolModal()`

and `installButtonPressed()` to `navigateToInstallPage()`